### PR TITLE
Rtp despawn player

### PR DESCRIPTION
**Pull request by MauxC**

### CODE REVIEW

File: src/systems/NetworkClientSystem.cpp [hunk 1]
```diff
@@ -66,6 +72,20 @@ namespace ecs
         _msgQueue.clear();
     }
 
+    void NetworkClientSystem::removePlayer(std::string s, SceneManager &sceneManager)
+    {
+        unsigned int id = std::stoi(s.erase(0, s.find(" ") + 1));
+        std::cerr << "Player " << id << " is being removed" << std::endl;
```
Comment (Varriss) [inline src/systems/NetworkClientSystem.cpp:78]: Could be removed

File: src/systems/NetworkServerSystem.cpp [hunk 2]
```diff
@@ -64,7 +70,7 @@ namespace ecs
                     continue;
                 auto pos = Component::castComponent<Position>((*p)[IComponent::Type::POSITION]);
                 playerComp->moveRight(manager, p, dt);
-                std::cerr << "Player pos: " << pos->x << ", " << pos->y << std::endl;
+                // std::cerr << "Player pos: " << pos->x << ", " << pos->y << std::endl;
```
Comment (Varriss) [inline src/systems/NetworkServerSystem.cpp:73]: Debug comment here

File: src/systems/NetworkServerSystem.cpp [hunk 3]
```diff
@@ -76,7 +82,7 @@ namespace ecs
                     continue;
                 auto pos = Component::castComponent<Position>((*p)[IComponent::Type::POSITION]);
                 playerComp->stopRight(manager, p, dt);
-                std::cerr << "Player pos: " << pos->x << ", " << pos->y << std::endl;
+                // std::cerr << "Player pos: " << pos->x << ", " << pos->y << std::endl;
```
Comment (Varriss) [inline src/systems/NetworkServerSystem.cpp:85]: Debug comment here

File: src/systems/NetworkServerSystem.cpp [hunk 4]
```diff
@@ -88,7 +94,7 @@ namespace ecs
                     continue;
                 auto pos = Component::castComponent<Position>((*p)[IComponent::Type::POSITION]);
                 playerComp->moveLeft(manager, p, dt);
-                std::cerr << "Player pos: " << pos->x << ", " << pos->y << std::endl;
+                // std::cerr << "Player pos: " << pos->x << ", " << pos->y << std::endl;
```
Comment (Varriss) [inline src/systems/NetworkServerSystem.cpp:97]: Debug comment here

File: src/systems/NetworkServerSystem.cpp [hunk 5]
```diff
@@ -127,7 +133,7 @@ namespace ecs
                     continue;
                 auto pos = Component::castComponent<Position>((*p)[IComponent::Type::POSITION]);
                 playerComp->stopUp(manager, p, dt);
-                std::cerr << "Player pos: " << pos->x << ", " << pos->y << std::endl;
+                // std::cerr << "Player pos: " << pos->x << ", " << pos->y << std::endl;
```
Comment (Varriss) [inline src/systems/NetworkServerSystem.cpp:136]: Debug comment here

File: src/systems/NetworkServerSystem.cpp [hunk 6]
```diff
@@ -153,7 +159,7 @@ namespace ecs
                     continue;
                 auto pos = Component::castComponent<Position>((*p)[IComponent::Type::POSITION]);
                 playerComp->stopDown(manager, p, dt);
-                std::cerr << "Player pos: " << pos->x << ", " << pos->y << std::endl;
+                // std::cerr << "Player pos: " << pos->x << ", " << pos->y << std::endl;
```
Comment (Varriss) [inline src/systems/NetworkServerSystem.cpp:162]: Debug comment here

**Review by CyrilDehaese — DISMISSED** [review]

**Review by Varriss — APPROVED** [review]

**Review by CyrilDehaese — APPROVED** [review]

**Pull request merged**